src/pages/CartPage.tsx

```diff
@@ -1,11 +1,18 @@
 /* eslint-disable max-len */
 import React from 'react';
+import { useSelector } from 'react-redux';
+import { RootState } from '../redux';
 import { Button } from '../components/Button';
-
-const totalSum = '2657';
-const itemsAmount = '3';
+import CartItem from '../components/CartItem';
+import { getTotalProductsCost } from '../utils/getTotalCost';
+import { getTotalItemsCount } from '../utils/getTotalItemsCount';
 
 export const CartPage: React.FC = () => {
+  const cartItems = useSelector((state: RootState) => state.cart.items);
+
```

Comment:
1. Don't use default useSelector, we have our own useAppSelector which already typified
2. (optional) destructure it like { items } from state.cart